src/Confluent.Kafka/Consumer.cs

```diff
@@ -184,6 +186,11 @@ public void Stop()
             remove { consumer.OnPartitionEOF -= value; }
         }
 
+        public event EventHandler<AggregateException> OnBackgroundPollException;
```

Comment:
i'm not sure this is the best name for this.

i've taken the approach that any exception that occurs on the background poll loop (e.g. in OnMessage etc) should be surfaced via an event on `Consumer`, currently called `OnBackgroundPollException`.

I think this works nicely and I can't think of any sane alternative, but welcome any feedback on this.

Note: had i thought through the complexities of correctly implementing a background poll loop on `Consumer`, i probably would have been in favor of omitting it from version 1...